dashboard/app/assets/stylesheets/application.scss

```diff
@@ -872,25 +853,42 @@ $small-footer-standard-width: 400px;
       .version-caption {
```

Comment:
`max-width: 211px;` on line 853 above (inside the `.small-footer-base` rule will make the footer show up small enough in music lab

![image](https://github.com/user-attachments/assets/9fe47c21-52e5-4004-a266-d9e9f1326fc3)